### DISCUSSION TITLE
Add a sort of album view that groups the songs on a playlist by album.

### DISCUSSION THREAD

**Discussion by TheMemoman** [audioling/audioling#9] [Ideas]
A way to show the songs that is more graphic, showing album covers that groups the songs by album.

This would be iTunes version of it:

![image](https://github.com/user-attachments/assets/8cad786d-b8e5-4551-a235-0145218b6643)

PS: Big fan of Feishin, best of luck with this new project!

**Comment by DanielNautre** [comment]
I would also love to see this as I'm often using Playlists as album collection
